CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
-# unrelease
+# 0.2.0 - 2024-02-27
 
+There are a bunch of breaking changes in this release, 
+
+- The `FromHex` trait has changed considerably. 
+- The `BufEncoder` got revamped.
+- A bunch of public errors have changed.
+
+More specifically:
```

Comment:
That makes it sound like all of the following changes are breaking which they are not.